samples/subsys/tracing/src/main.c

```diff
@@ -7,6 +7,10 @@
 
 #include <zephyr.h>
 #include <sys/printk.h>
+#include <logging/log.h>
+#if (defined(CONFIG_USB) && defined(CONFIG_USB_DEVICE_STACK))
```

Comment:
> I do think we need a `cdc_acm_uart_ready()` API function and that the examples should use it. The console in particular will not work well if material gets output before it's ready.
> 
> IMO it shouldn't be `uart_console_ready()` because it's not generic to uart, and it's not specific to console. It is generic to CDC_ACM.

By this patch I don't want to address the #27160 but just make `usb_enable()` be called across all the samples that use USB.

All other comments are addressed.

Reply:
I've removed all conditional includes.